pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler_test.go

```diff
@@ -641,6 +643,128 @@ func TestCreateVirtualService(t *testing.T) {
 					},
 				},
 			},
+		}, {
+			name: "found predictor status with path template and the additional ingress domains",
```

Comment:
path template does not need the additional domains, it is only for host based routing